### PR TITLE
Nexpose add scan

### PR DESCRIPTION
**Pull request by liorblob**
solves https://github.com/demisto/etc/issues/12369

### CODE REVIEW

File: Playbooks/playbook-Scan_Nexpose_Assets.yml [hunk 1]
```diff
@@ -0,0 +1,206 @@
+id: 1a457659-b9b4-47f4-853d-67a34e4b0784
```
Comment (anara123) [inline Playbooks/playbook-Scan_Nexpose_Assets.yml]: Change the id to be readable

File: Playbooks/playbook-Scan_Nexpose_Assets.yml [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+version: 10
```
Comment (anara123) [inline Playbooks/playbook-Scan_Nexpose_Assets.yml]: change the version to be -1

**Comment by anara123** [comment]
Attach screenshot of the playbook

**Comment by liorblob** [comment]
Assets scan sub-playbook:
nexpose_scan_assets_wed_aug_01_2018
Site scan sub-playbook:
nexpose_scan_site_wed_aug_01_2018

**Pull request merged**